autopostgresqlbackup

```diff
@@ -211,7 +211,7 @@ fi
 # Hostname for LOG information and
 # pg_dump{,all} connection settings
 if [ "${DBHOST}" = "localhost" ]; then
-    HOST="$(hostname --fqdn)"
+    HOST="$(uname -n)"
```

Comment:
I'd prefer to keep the FQDN hostname (instead of only the node name). We should find a portable way to get it. The use of `domainname` does not seems to be an option.

Reply:
@dennisse I propose you to remove the hostname related changes and move them to another PR so that we could discuss the best to handle this. Once done I will merge your portability changes if you agree?